tests/unit/htdocs/class/cache/XoopsCacheTest.php

```
@@ -659,23 +667,46 @@ public function testWriteWithLargeDataStructure()
+            $this->assertIsArray($config1);
+            $this->assertIsArray($config2);
+            $this->assertEquals('file', $config1['engine']);
+            $this->assertEquals('file', $config2['engine']);
+            $this->assertNotEquals(
+                $config1['settings']['path'] ?? null,
+                $config2['settings']['path'] ?? null,
+                'Two engine configs should have distinct paths'
+            );
+        } finally {
+            // Recursively remove test directory and any cache files created by the engine
+            $entries = new \RecursiveIteratorIterator(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '695', column '28'). (undefined)

(MissingImport)